addons/medical/Language/ACE_Medical_localization.st

```diff
@@ -11,6 +11,13 @@ StringTable {
    Author "kexanone"
    LastChanged "apollo-jk"
   }
+  CustomStringTableItem "{61D51487241F1A28}" {
+   Id "ACE_Medical-FailReason_CriticallyInjured"
+   Target_en_us "Too injured - Requires medical facility"
```

Comment:
Make it the same as the message that is shown for the medical kit
```suggestion
   Target_en_us "Too injured (Requires a field hospital or ambulance)"
```

Reply:
Localization changes to come

Reply:
Updated message to match the medical kit